massa-versioning/src/versioning_ser_der.rs

```diff
@@ -77,12 +71,6 @@ impl Serializer<MipInfo> for MipInfoSerializer {
 
         // Components
         let components_len_ = value.components.len();
-        if components_len_ > MIP_INFO_COMPONENTS_MAX_ENTRIES as usize {
-            return Err(SerializeError::NumberTooBig(format!(
-                "MIP info cannot have more than {} components, got: {}",
-                MIP_STORE_MAX_ENTRIES, components_len_
-            )));
-        }
         let components_len = u32::try_from(components_len_).map_err(|_| {
             SerializeError::GeneralError(format!(
                 "Cannot convert to component_len: {} to u32",
```

Comment:
same as above `to` -> `from` (I think)

Reply:
And the parameter to the `format!` should be `compenets_len` (not `name_len_` as it is currently.

Reply:
Fixed